src/nvim/screen.c

```diff
@@ -7102,11 +7102,13 @@ void screen_resize(int width, int height)
         update_topline();
         if (pum_drawn()) {
           redraw_later(NOT_VALID);
-          ins_compl_show_pum();           /* This includes the redraw. */
-        } else
+          ins_compl_show_pum();
+        } else {
           update_screen(NOT_VALID);
```

Comment:
in the [vim patch](https://github.com/vim/vim/commit/a5e6621aadadf78c7b344e93a4b328788076f14c?w=1) this `else` was removed, so that `update_screen(NOT_VALID)` is always called. (The vim commit message hints about this: "also when a popup menu is visible").